### DISCUSSION TITLE
How to find the watch tower clients connected to your node

### DISCUSSION THREAD

**Discussion by BhaagBoseDK** [lightningnetwork/lnd#5678] [Q&A] [answered]
I run a watch tower. How can I see
1- Data and Bandwidth usage consumed by the watch tower clients connecting to me.
2- Who is connecting to my watch tower.
3- Any way to disable/remove a particular watch tower client from connecting to me.

**Comment by guggero** [comment, marked as answer]
There's the `lncli tower info` command. That is currently all the information there is about a tower you're running. Also there's no way to remove or ban clients at the moment.

**Comment by BhaagBoseDK** [comment]
lncli tower info only shows info from my node point of view. I get no idea who and how many clients are connecting to my node.

`umbrel@umbrel:~/charge-lnd $ ~/umbrel/bin/lncli tower info


{
    "pubkey": "XXXXX",
    "listeners": [
        "[::]:9911"
    ],
    "uris": [
        "xxxxxxx@yyyyy.onion:9911"
    ]
}`
